File: C++/0x03/1475.cpp
#include <bits/stdc++.h>//자주 사용 라이브러리 전부 컴파일
using namespace std;//std 네임 스페이스를 사용
//방번호

int main(void){
  int a,num[10]={0,},ans=1;
  ios::sync_with_stdio(0);
  cin.tie(0);
  cin>>a;
  while(a){
    num[a%10]++;
    a/=10;
  }
  for(int i=0;i<10;i++){
    if(i==6 || i==9) continue;
    ans=max(ans,num[i]);//1,4
  }//ans=4
  ans=max(ans,(num[6]+num[9]+1)/2);//4,2
  cout<<ans;
  
}

File: Python/2022_3/1475.py
#방번호
n=input()
num=[0]*10
mix=0

for i in n:
    if(i=='6' or i=='9'):
        mix+=1
    else:
        num[int(i)]+=1

if(mix%2==0):
    mix=mix//2
else:
    mix=mix//2+1
num[6]=mix
print(max(num))
